internal/pkg/sylog/sylog.go

```diff
@@ -5,6 +5,9 @@
 
 // Package sylog implements a basic logger for Singularity Go code to log
```

Comment:
I think it would be smart here to create a new file `doc.go`, which is just an empty file with the package comment. This way we don't have to worry about the package comment not displaying in `go doc` when `sylog` build tag isn't set in the environment. Something like this [`doc.go`]:

```
// Package sylog implements a basic logger for Singularity Go code to log
// messages in the same format as singularity_message() from C code
package sylog
```

Reply:
Then we can also remove the package comment from this `sylog.go` file